backend-go/Makefile

```diff
@@ -0,0 +1,28 @@
+CODEPAIR_VERSION := 0.2.0
+
+default: help
+
+tools: ## install tools for developing codepair
+	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.55.1
+
+build: ## builds an executable that runs in the current environment
+	go build -o ./bin/codepair cmd/codepair/main.go
```

Comment:
How about only specifying only the [package level](https://pkg.go.dev/cmd/go#hdr-Compile_packages_and_dependencies)?

- `go build [-o output] [build flags] [packages]`
- `go build -o ./bin/codepair ./cmd/codepair`